internal/linker/linker.go

```diff
@@ -137,6 +137,8 @@ func applyPatches(srcDir, workingDir string, modFiles map[string]bool, patches [
 	return mod, nil
 }
 
+// TODO: put linker binaries into fsCache in the main package
```

Comment:
how will it work with cache trimming?

Reply:
We trim _after_ a build, and any file not used in the past five days is removed. So, in theory, at least one linker binary should stick around.